app/models/concerns/thredded/topic_common.rb

```diff
@@ -10,7 +10,7 @@ module TopicCommon
                  foreign_key: 'last_user_id'
 
       scope :order_recently_updated_first, -> { order(updated_at: :desc, id: :desc) }
-      scope :on_page, -> (page_num) { page(page_num).per(30) }
+      scope :on_page, -> (page_num) { page(page_num) }
```

Comment:
the `paginates_per 50` is tied to a conditional. In most/all cases - does that conditional resolve to true?

1. I'm wondering if that conditional can be removed if such is the case.
2. If it wavers between true/false I'm guessing the `per(30)` is to lock it to a default?

In either case, you're right. It's a little confusing to have two approaches 7 lines apart with anything stating why